README.md

```diff
@@ -118,17 +118,18 @@ The following sections provide a full list of configuration in- and output varia
 <a id="markdown-Inputs" name="Inputs"></a>
 
 | Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:-----:|
+|------|-------------|------|---------|:--------:|
 | apex\_domain | The main domain to either use directly or to configure a subdomain from | `string` | `""` | no |
 | cluster\_name | Variable to provide your desired name for the cluster. The script will create a random name if this is empty | `string` | `""` | no |
-| cluster\_version | Variable to provide your desired Kubernetes version for the cluster. | `string` | `"1.15"` | no |
+| cluster\_version | Kubernetes version to use for the EKS cluster. | `string` | `"1.15"` | no |
```

Comment:
For what it's worth, there is a `make markdown-table` which used `terraform-docs` to create this table. The Make rule prints the Markdown to the console which you then can cut&paste into the README. Not ideal, but it helps a bit.

Where I am getting is that there is a risk that if you only update this table without the underlying documentation in the variables, there is a risk this change will disappear when someone else uses `make markdown-table` again.

Do you mind updating?

Reply:
done.